web/locales/en/messages.ftl

```diff
@@ -218,7 +218,7 @@ sc-criteria-item-1 = The sentence must be spelled correctly.
 sc-criteria-item-2 = The sentence must be grammatically correct.
 sc-criteria-item-3 = The sentence must be speakable.
 sc-criteria-item-4 = If the sentence meets the criteria, click the &quot;Approve&quot; button on the right.
-sc-criteria-item-5 = If the sentence does not meet the above criteria, click the &quot;Reject&quot; button on the right. If you are unsure about the sentence, you may also skip it and move on to the next one.
+sc-criteria-item-5 = If the sentence does not meet the above criteria, click the &quot;Reject&quot; button on the left. If you are unsure about the sentence, you may also skip it and move on to the next one.
```

Comment:
I feel this would warrant triggering a re-translation. What do you think? If you agree, let's change the key (both here and where it's used in the component) to trigger that. Firefox is using `-2` prefixes if there are no clear other changes, so I'd go with `sc-criteria-item-5-2`, as odd as it is.

Reply:
Should be OK now.